spec.bs

```diff
@@ -107,18 +108,29 @@ dictionary AuctionAdInterestGroup {
 
 <div algorithm="joinAdInterestGroup()">
 
-The <dfn for=Navigator method>joinAdInterestGroup(|group|, |durationSeconds|)</dfn> method steps
-are:
+The <dfn for=Navigator method>joinAdInterestGroup(|group|)</dfn> method steps are:
+<div class="note">
+
+Note: Temporarily, Chromium does not include the <span class="allow-2119">required</span> keyword
+for {{AuctionAdInterestGroup/lifetimeMs}}, and instead starts this algorithm with the step
+
+1. If |group|["lifetimeMs"] does not [=map/exist=], throw a {{TypeError}}.

```

Comment:
You changed several of these to monospace in another PR, but where there's a definition, it's probably better to link to that instead.

```suggestion
1. If |group|["{{AuctionAdInterestGroup/lifetimeMs}}"] does not [=map/exist=], throw a {{TypeError}}.
```

Reply:
sorry, do you have a link to the changes? I cannot remember that I had done that.

Maybe [these monospace changes ](https://github.com/WICG/turtledove/pull/610/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR1234)?
These are fields of ordered maps, which do not have definitions.

Reply:
Yep, #610. Nothing's wrong with that change; I was just thinking about how to be consistent with it here.